Dockerfile.cross

```diff
@@ -0,0 +1,16 @@
+# This image is meant to enable cross-architecture builds.
+# It assumes the ream binary has already been
+# compiled for `$TARGETPLATFORM` and moved to `./bin`.
```

Comment:
This is not true. L13 assumes the ream binary is located at `./dist/bin/$TARGETARCH`.